src/app/shared/components/duration-input/duration-input.component.html

```diff
@@ -1,11 +1,12 @@
 <div class="duration-input" [formGroup]="durationFormGroup">
```

Comment:
I don't understand why it's form group again. You don't have several inputs here.

Reply:
fixed